discovery/chan_series.go

```diff
@@ -217,6 +217,16 @@ func (c *ChanSeries) FilterChannelRange(chain chainhash.Hash,
 
 	chanResp := make([]lnwire.ShortChannelID, 0, len(chansInRange))
 	for _, chanID := range chansInRange {
+		// Maybe this check should live inside the previous
+		// graph.FilterChannelRange?
```

Comment:
`FilterChannelRange` indeed sounds like a good place for this check 👍

Reply:
Updated in the fixup commit. Also added a change to the relevant unit test.